testing/src/main/java/org/oppia/android/testing/AssertionHelper.kt

```diff
@@ -0,0 +1,24 @@
+package org.oppia.android.testing
+
+import org.junit.Assert.fail
+import kotlin.reflect.KClass
+import kotlin.reflect.full.cast
+
+object AssertionHelper {
```

Comment:
Can we a Kdoc here to understand what this file is and when to use it!!

Reply:
Note that the documentation should define what kinds of helpers we want to put in this file (in general, they should always be really generic like this assertThrows--we should never add assertion helpers here that are specific to a certain test or part of the codebase. Those should go in the individual test suite or module-specific test libraries).

Reply:
Added.